components/SearchResultsList.vue

```diff
@@ -10,24 +10,35 @@
+                        $t('searchResultsList.filters') }}</span>
             </button>
         </div>
-        <div id="searchResults" class="flex flex-col overflow-y-scroll h-full">
-            <div @click="resultsStore.setActiveSearchResult(searchResult.professional.id)" :key="index"
-                v-for="(searchResult, index) in resultsStore.searchResultsList" class="results-list flex flex-col">
-                <div class="flex-1 h-24 w-6/8 border-b-2 border-primary/20 p-3
-                    hover:border-transparent hover:bg-primary/5 transition-all hover:cursor-pointer">
-                    <SearchResultsListItem
-                        :name="`${searchResult.professional.names[0].firstName} ${searchResult.professional.names[0].lastName}`"
-                        :degrees="searchResult.professional.degrees"
-                        :facility-name="localeStore.locale?.code == Locale.EnUs ? searchResult.facilities[0]?.nameEn : searchResult.facilities[0]?.nameJa"
-                        :specialties="searchResult.professional.specialties"
-                        :spoken-languages="searchResult.professional.spokenLanguages" />
+        <template v-if="resultsStore.searchResultsList.length > 0">
+            <div id="searchResults" class="flex flex-col overflow-y-scroll h-full">
+                <div @click="resultsStore.setActiveSearchResult(searchResult.professional.id)" :key="index"
+                    v-for="(searchResult, index) in resultsStore.searchResultsList" class="results-list flex flex-col">
+                    <div class="flex-1 h-24 w-6/8 border-b-2 border-primary/20 p-3
+                        hover:border-transparent hover:bg-primary/5 transition-all hover:cursor-pointer">
+                        <SearchResultsListItem
+                            :name="`${searchResult.professional.names[0].firstName} ${searchResult.professional.names[0].lastName}`"
+                            :degrees="searchResult.professional.degrees"
+                            :facility-name="localeStore.locale?.code == Locale.EnUs ? searchResult.facilities[0]?.nameEn : searchResult.facilities[0]?.nameJa"
+                            :specialties="searchResult.professional.specialties"
+                            :spoken-languages="searchResult.professional.spokenLanguages" />
+                    </div>
                 </div>
             </div>
-        </div>
+        </template>
+        <template v-else>
+            <div class="h-full flex flex-col w-[358px]">
+                <img src="../assets/images/No-search-results-graphic.png" class="p-12" />
+                <span class="text-primary-text pl-1 font-bold self-center group-hover:text-primary-text-inverted/50">No
+                    Results Found</span>
+                <span class="text-primary-text self-center">Please try a different filter comination</span>
```

Comment:
Let's change this to a translatable string using `$t('')` and adding it to the files in the /locales directory.

Reply:
Also, please fix the spelling for "combination" 🙏🏾

Reply:
You got it.